js/stdlib/ts/ALGO.ts

```diff
@@ -2707,6 +2707,9 @@ const launchToken = async (accCreator: Account, name: string, sym: string, opts:
   return { name, sym, id, mint, optOut };
 }
 
+const contract = (bin: Backend, ctcInfo?: Promise<ContractInfo>): Promise<Contract> =>
+  createAccount().then(acc => acc.contract(bin, ctcInfo));
```

Comment:
I think there should be something in `shared_impl` to make it easy to do this and keep them in all in sync

Reply:
i did this, but i don't think it's better